packages/server/graphql/mutations/setOrgUserRole.ts

```diff
@@ -38,8 +38,13 @@ export default {
 
     // AUTH
     const viewerId = getUserId(authToken)
-    if (!(await isUserBillingLeader(viewerId, orgId, dataLoader, {clearCache: true}))) {
-      return standardError(new Error('Must be the organization leader'), {userId: viewerId})
+    if (
+      !isSuperUser(authToken) ||
+      !(await isUserBillingLeader(viewerId, orgId, dataLoader, {clearCache: true}))
```

Comment:
If I **AM** a super user but **NOT** the billing leader for the org, this will still be true and I can't run the mutation

Reply:
oh. this should not have happened. thank you!